src/utils/helperTransition/index.tsx

```diff
@@ -38,16 +40,16 @@ const getStyleForTransitionState = (transitionState: string): object => {
 
 type HelperTransitionParamsType = {
   children: JSX.Element;
-  props: Array<any>;
+  [key: string]: any;
 };
 
 const HelperTransition = ({
   children: child,
   ...props
-}: HelperTransitionParamsType): React.ReactNode => (
+}: HelperTransitionParamsType) => (
```

Comment:
actually removing return types and let TS infer them solved a bunch of TS errors